### PR TITLE
Update OpenTelemetry and resolve few deprecations

### PR DESCRIPTION
**Pull request by im-konge**
This PR updates OpenTelemetry, removes things that are not needed and also deals with few deprecations that were present here for a while.

### CODE REVIEW

**Review by jankalinic — APPROVED** [review]

**Review by see-quick — APPROVED** [review]

**Comment by ppatierno** [comment]
LGTM but I would wait for the same stuff merged into the bridge and operator before merging this one.

**Pull request merged**
